dev-python/sentinels/sentinels-1.0.0.ebuild

```diff
@@ -0,0 +1,25 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{6,7,8} )
+
+inherit distutils-r1
+
+DESCRIPTION="Python Sentinels class which are objects with special meanings."
+HOMEPAGE="https://pypi.org/project/sentinels"
+SRC_URI="mirror://pypi/${PN:0:1}/${PN}/${P}.tar.gz"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="test"
```

Comment:
not needed,
distutils_enable_tests defines it for you.

Reply:
removed